### ISSUE TITLE
Square brackets in issue title break the markdown rendition

### ISSUE THREAD

**Issue by retailcoder** [Zomis/Duga#113]
![image](https://cloud.githubusercontent.com/assets/5751684/11431220/28e2fd2e-9461-11e5-959e-91126edc3c94.png)

Escaping them with a backslash would fix this.

**Comment by Zomis** [comment]
Work around: Don't use square brackets ;)

**Comment by retailcoder** [comment]
😒 ... right 😝

**Comment by Zomis** [comment]
@retailcoder Is this and #63 the same issue? (If it is, I will close #63 as I think this description is better)

**Comment by Vogel612** [comment]
Yep that's the same thing. Since message splitting doesn't even apply here, my comment there doesn't apply anyhow.
